keyboards/kprepublic/bm60hsrgb_iso/rev2/rules.mk

```diff
@@ -0,0 +1,38 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
```

Comment:
?

Reply:
?

Reply:
There's no bootloader specified.

Reply:
What should I do?

Reply:
Specify one? I'm guessing it's probably `atmel-dfu`, but you should double check.

Reply:
<img width="626" alt="image" src="https://user-images.githubusercontent.com/43021794/143828666-0d2db28d-88ee-46d6-ade7-36c225532487.png">
facepalm.jpg

Reply:
fixed